packages/web/src/components/common/reward-tooltip-content/RewardTooltipContent.tsx

```diff
@@ -128,100 +128,57 @@ const RewardTooltipContent: React.FC<
     });
   }, [externalRewards]);
 
+  const rewardsData = [
+    { type: "SWAP_FEE", rewards: swapFeeRewards, totalUSD: swapFeeRewardUSD },
+    { type: "INTERNAL", rewards: internalRewards, totalUSD: internalRewardUSD },
+    { type: "EXTERNAL", rewards: externalRewards, totalUSD: externalRewardUSD },
+  ]
+    .filter(({ rewards }) => rewards)
+    .sort((a, b) => {
+      const totalA = parseFloat(a.totalUSD.replace(/[$, ]/g, ""));
+      const totalB = parseFloat(b.totalUSD.replace(/[$, ]/g, ""));
```

Comment:
It is possible for the value of usd to be `-`.
Therefore, the values of totalA, totalB can be `NaN`.